saleor/graphql/core/types/common.py

```diff
@@ -102,6 +102,10 @@ class StockError(Error):
     code = StockErrorCode(description="The error code.", required=True)
 
 
+class BulkStockError(ProductError):
+    id = graphene.ID(description="ID of input element which cause the error.")
```

Comment:
In `BulkProductError` we use `index`; I think it would be better to do it here as well.